azurelinuxagent/common/protocol/util.py

```diff
@@ -186,7 +158,7 @@ def get_wireserver_endpoint(self):
 
         return self.endpoint
 
-    def _set_wireserver_endpoint(self, endpoint):
+    def set_wireserver_endpoint(self, endpoint):
```

Comment:
Let's leave this one marked as private and let the migration utils reach into the private.
Idea is that this method should be directly called from code outside of protocol/*.
Same for save_protocol